pkg/network/modules.go

```diff
@@ -63,12 +63,14 @@ type Port struct {
 	PortName   string
 	Virtual    bool
 	DPDK       bool
+        UnixSocket bool
 	IfaceName  string
 	Network    string
 	NamesSpace string
 	IPAddr     string
 	MacAddr    string
 	Gateway    string
+        UnixPath   string //used in Socket Port and DPDK vhost-user
```

Comment:
In the DPDK patch I added SocketPath field:
https://github.com/nimbess/nimbess-agent/pull/11/commits/5fa5e7c1860723363198c3fbd73fec900cde4286#diff-4cee0960744d01b5f4d7354d8b48b8b0R72

Can you please  these earlier commits on top of that and re-use that var?

Reply:
It is reused in the latest version - note "outdated" on the github interface.